web/src/hotkey.js

```diff
@@ -1316,31 +1315,17 @@ export function process_hotkey(e, hotkey) {
     return false;
 }
 
-/* We register both a keydown and a keypress function because
-   we want to intercept PgUp/PgDn, Esc, etc, and process them
-   as they happen on the keyboard. However, if we processed
-   letters/numbers in keydown, we wouldn't know what the case of
-   the letters were.
-
-   We want case-sensitive hotkeys (such as in the case of r vs R)
-   so we bail in .keydown if the event is a letter or number and
-   instead just let keypress go for it. */
-
 export function process_keydown(e) {
     activity.set_new_user_input(true);
-    const hotkey = get_keydown_hotkey(e);
-    if (!hotkey) {
+    const result = get_keydown_hotkey(e);
```

Comment:
Probably ideally we should be coming up with a name for the type of the objects that `get_keydown_hotkey` returns and referring to that here rather than the generic `result`, which would usually be a good name for something we'll return; we can defer that for when we do the TS migration for this module, I guess.

Reply:
Yeah, I spent a few minutes to figure out a better variable name -- which indicates that it can be a single hotkey or an array. Ended up with a very generic `result`.